### DISCUSSION TITLE
Projects Beta: Remove Save changes button and save all the time

### DISCUSSION THREAD

**Discussion by mikela** [community/community#13444] [Projects and Issues]
Hi GitHub, I'm creating a new view in the new Projects Beta feature and I think you should just save changes all the time. The _Save changes_  button only adds friction. I can easily duplicate an existing view and experiment with that if I need to. Thank you.

**Comment by BirgitPohl** [comment]
I would disagree. 
My team has a good set of filters, but sometimes they temporarily adjust the settings and don't want them to be saved for everyone. If they really require to have it saved for everyone, then they intentionally press the save button.

**Reply by mikela** [reply]
You might be interested in "Personal views" https://github.com/github/feedback/discussions/6423

**Reply by BirgitPohl** [reply]
It's interesting, but not necessarily what I was saying.
Let me be a bit more clear: While screen sharing the team view, that everyone is familiar with, the team is temporarily adjusting the settings. For example, to find an item that has disappeared through all the filtering. And you might know yourself, that the current way of sorting the items can be quite confusing. They never had the intention to save that.
